src/ui/app/package.json

```diff
@@ -10,11 +10,15 @@
     "lint:fix": "eslint '*/**/*.{js,ts,tsx}' --format table --fix"
   },
   "dependencies": {
+<<<<<<< HEAD
```

Comment:
note to clean up these

Reply:
Oops I didn't have my merge conflict fixes. Thanks for pointing them out!